### DISCUSSION TITLE
Request : Maintenance - Remove all existing Lane Connectors

### DISCUSSION THREAD

**Discussion by DanielCoffey** [CitiesSkylinesMods/TMPE#1637] [Ideas]
Feature Request : Add a new button to the in-game TM:PE Options - Maintenance tab labelled "Remove all existing Lane Connectors".

I would like to ask for a way to remove all Lane Connectors from every Node on an existing map.

When loading someone else's Save File for an attempt at a City Fix, we often encounter Lane Connector spaghetti. This causes unexpected traffic flow at junctions and roundabouts. Often when a Builder has been struggling with increasing trafic, many Lane Connectors are left all over the map as they try to fix their issues. When needing to start over with the traffic flow, these connectors are a nuisance and can be hard to spot when not zoomed in enough. At the moment the only way to remedy this is to remove the connectors from each Node individually. An automated tool to zap all the Lane Connectors at once would be a much appreciated idea.

**Comment by krzychu124** [comment]
I assume that a button somewhere at the Maintenance tab would be enough for it, right? If so, I'll try to implement it and see if it's possible also for other tools.

**Reply by DanielCoffey** [reply]
That would be very much appreciated, yes. I feel for certain that the "City Fix" youtubers would find this feature useful.
